.github/workflows/build-test.yaml

```diff
@@ -92,34 +87,27 @@ jobs:
       shell: bash
       env:
         dependencies: ${{ matrix.dependencies }}
-      run: |
-        python -m venv .venv
-        source .venv/${{matrix.venv-loc}}/activate
-        source ./tests/ci_install.sh
+      run: source ./tests/ci_install.sh
 
     - name: Install and patch nosetest
       if: matrix.test-runner == 'nose'
+      # note: this could be handled with [tool.uv.sources]
```

Comment:
to add to this comment: the idea is we could host a fork of nose within our org, apply this patch, and use the fork as the source instead of PyPI. This would only work with uv but would arguably less of a hack. Or we could also try to use `nose2` ...